docs/reference-guides/rancher-webhook.md

```diff
@@ -38,3 +58,34 @@ When using a private GKE cluster, errors may occur that prevent the Kubernetes A
 ```
 This issue occurs because firewall rules restrict communication between the API server and the private cluster. To resolve this communication problem, users must add firewall rules to allow the GKE control plane to communicate with the Rancher-Webhook on port 9443. Please refer to the [GKE documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules) for detailed information and steps on updating the firewall rules.
+
+### Application Fails to Deploy due to rancher-webhook blocking access
+
+The webhook provides extra validations on [namespaces](https://github.com/rancher/webhook/blob/release/v0.4/docs.md#psa-label-validation). One of these validations ensures that users can only update PSA relevant labels if they have the proper permissions (`updatepsa` for `projects` in `management.cattle.io`). This can result in specific operators (such as tigera or trident) which attempt to deploy namespaces with PSA labels failing to deploy these resources. There are several ways that this issue can be resolved:
+
+- The application (e.g. tigera/trident) should be configured to create a namespace with no PSA permissions/labels. If users which to apply a PSA to these namespaces, they can add them to a project with the desired PSA after configuration. See the [docs on PSS and PSA resources](../../how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards) for more information on how to perform this operation.
+  - This is the preferred option, though not all applications can be configured in this fashion.
+- Users can manually grant the operator permissions to manage PSAs for namespaces.
+  - This option will introduce new security risks, since this operator will now be able to set the PSA for the namespaces it has access to. This could allow the operator to deploy a privileged pod (or effect cluster takeover through other means).
+- A User with the proper permissions can pre-create the namespace with the appropriate configuration.
+  - This option depends on the ability of the application to handle existing resources.
+
+## Issues on Specific Versions
+
+**Note:** This section contains an incomplete listing of high-severity issues affecting specific Rancher/webhook versions. In most cases, these can be resolved by upgrading to a more recent Rancher version.
+
+### Incompatible webhook version on Rollback
+
+**Note:** This affects rolling back to Rancher versions prior to `v2.7.5`
+
+Rancher users who rollback to a version of rancher before `v2.7.5` may see webhook versions higher than is acceptable for the pre-2.7.5 version of Rancher in downstream clusters. This may lead to various incompatability issues (such as the issue where Project Members can't create namespaces). In addition, when rolling back to versions before the webhook was installed in downstream clusters, the webhook may remain installed, which can result in similar compatability issues. 
```

Comment:
```suggestion
If you rollback to Rancher v2.7.5 or earlier, you may see webhook versions that are too recent to be compatible with downstream clusters running pre-v2.7.5 version of Rancher. This may cause various incompatibility issues. For example, project members may be unable to create namespaces. In addition, when you roll back to versions before the webhook was installed in downstream clusters, the webhook may remain installed, which can result in similar incompatibility issues. 
```

Reply:
In the suggested change, `rollback` -> `roll back` in the first sentence